lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -7089,7 +7090,7 @@ namespace Js
     {
 #if ENABLE_PROFILE_INFO
         // Switch off profiling is asmJsFunction
-        if (this->GetIsAsmJsFunction())
+        if (this->GetIsAsmJsFunction() || this->GetAsmJsModuleInfo())
```

Comment:
looks not, GetIsAsmjsMode is not for AsmJsModuleInfo. we have m_isAsmJsFunction, but don't have that for AsmJsModuleInfo